go-controller/pkg/node/gateway_localnet_linux_test.go

```diff
@@ -852,6 +852,10 @@ var _ = Describe("Node Operations", func() {
 					Cmd: "ovs-ofctl show ",
 					Err: fmt.Errorf("deliberate error to fall back to output:LOCAL"),
 				})
+				fakeOvnNode.fakeExec.AddFakeCmd(&ovntest.ExpectedCmd{
```

Comment:
So I noticed when running these tests, some of them error because the number of calls do not match, but interestingly it doesnt fail the test case. I think it's because the tests are missing:
Expect(fakeOvnNode.fakeExec.CalledMatchesExpected()).To(BeTrue(), fExec.ErrorDesc)

Anyway, I just added this one here because I noticed it while testing this particular test. If you want me to remove it I can.

Reply:
nah not a big deal. just checking its intended.